src/NUnitFramework/framework/Constraints/ComparisonConstraint.cs

```diff
@@ -143,11 +143,43 @@ public ComparisonConstraint Percent
             get
             {
                 _tolerance = _tolerance.Percent;
-                Description += " percent";
                 return this;
             }
         }
 
         #endregion
+        
+        #region Protected Methods
+        
+        /// <summary>
+        /// Provides standard description of what the constraint tests
+        /// based on comparison text.
+        /// </summary>
+        /// <param name="comparisonText">Describes, what constraint tests, throws <see cref="ArgumentNullException"/>
```

Comment:
Stray comma in here I forgot to comment on earlier. How about: `Describes the comparison being tested,`?

Reply:
That sounds smoother to me. English isn't my native, so any language advices are appreciated